canonical-web-and-design/ubuntu.com
# Revert archive takeovers page

## Done

- Revert archive takeovers page and move it to `takeovers-archive`

## QA

- Check demo/takeovers
- Check demo/takeovers-acrhive


## Issue / Card

Fixes #9430
